identity/app/services/PlaySavedArticlesService.scala

```diff
@@ -20,7 +21,7 @@ class PlaySavedArticlesService @Inject()(api: IdApiClient) extends SafeLogging w
   def getOrCreateArticlesList(auth: Auth): Future[Response[SavedArticles]] = {
     val savedArticlesResponse = api.savedArticles(auth)
     savedArticlesResponse.flatMap {
-      case Right(_) => savedArticlesResponse
+      case Right(response) => Future { Right(response.sanitizeArticleData)}
```

Comment:
The return type of this function seems to be causing problems, hence the need to reconstruct a `Future` `Right` object